### ISSUE TITLE
parse error - @charset

### ISSUE THREAD

**Issue by shin820** [kalidasya/sonar-css#7]
Hi, 
I got the following error when parsing @charset "utf-8";

18:05:13.045 ERROR - Unable to parse file: D:\Refactoring_Test\test\01
test\test_css\mission-css.css
com.sonar.sslr.api.RecognitionException: Parse error at line 1 column 2 failed t
o match all of: whiteSpaces lBracket idSelector classSelector colon lBracket idS
elector classSelector colon _ident from to _num adjacentComb precededComb childC
omb lBracket idSelector classSelector colon _ident from to _num adjacentComb pre
cededComb childComb comma eof comma lCurlyBracket statement

1: ?@charset "utf-8";
    ^

Failed at rules:
                /-whiteSpaces
              /-ident consumed from (1, 1) to (1, 1): "?"
            /-typeSelector
            |     /-lBracket
            |   /-attributeSelector
            |   +-idSelector
            |   +-classSelector
            |   | /-colon
            |   +-pseudo
            | /-subS
            +-typeSelector consumed from (1, 1) to (1, 1): "?"
          /-simpleSelector

**Comment by spil-tamas** [comment]
I am not a css expert, but what is that ? before the charset? it should be just: @charset "UTF-8";

**Comment by shin820** [comment]
Actually there is no sign '?' in my CSS file, but it appears when CSS parsing.

**Comment by kalidasya** [comment]
hm... perhaps, its a char encoding issue... what is the encoding setting in the sonar-runner.properties? can you check is the file properly encoded in UTF8?

**Comment by shin820** [comment]
Yes, this is a encoding issue.  The issue is result from that I'm a windows user and I used notepad to save this file as 'UTF8'.
About the details, please see http://koti.mbnet.fi/akini/java/unicodereader/

**Comment by kalidasya** [comment]
ok I will check can I do something from the plugin's side, thanks for the link!

**Comment by kalidasya** [comment]
can you try to reproduce this with the javascript plugin? so analyse a similar js file?
